Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+
+
+def validate_hmac_sha256(key: str, comment_w_hmac: str, separator="hmacsha256:"):
+    """
+
+    :param key:
+    :param comment_w_hmac: Resolved, from: ServiceNow Thu Nov 10 2022 07:56:13 GMT-0800 (PST),
+    close_code: Solution provided, close_notes: ddddddddddddd hmacsha256:
+    a9d7947047c371a10a44d07b381152951c46b0c954f8ddf1eeefc29fd120b8ae
+    :param separator:
+    :return:
+    """
+    result = [piece.strip() for piece in comment_w_hmac.split(separator)]
+    if len(result) != 2:
+        return False
+    comment, hash_mac = result
+    # logger.debug(f'{validate_hmac_sha256.__qualname__} comment: {comment}, {hash_mac}')
+    if get_hmac_sha256(key, comment) == hash_mac:
+        # logger.info(f'{validate_hmac_sha256.__qualname__} validating hmac is True: {comment}, {hash_mac}')
+        return True
+    return False
+
+
+def is_msg_from_third_party(key, comment):
+    if comment and validate_hmac_sha256(key=key, comment_w_hmac=comment):
+        return True
+    return False
+
+
+def add_prefix_to_comment(commnet: Optional[str]) -> str:
+    return f"from XSOAR Cortex {datetime.today():%Y%m%d_%H%M%S} {commnet},"
+
+
+def retrieve_labels_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+
+    response = client.retrieve_labels_request(page, items)
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveLabels",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_a_specific_label_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    label_id = args.get("label_id")
+
+    response = client.retrieve_a_specific_label_request(label_id)
+
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveASpecificLabel",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
+    )
+
+    return command_results
+
+
+def retrieve_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    page = args.get("page")
+    items = args.get("items")
+    fromdate = args.get("fromdate")
+    todate = args.get("todate")
+    status = args.get("status")
+    adversary_types = args.get("adversary-types")
+    labels = args.get("labels")
+
+    if status and isinstance(status, str):
+        status = status.split(",")
+    if adversary_types and isinstance(adversary_types, str):
+        adversary_types = adversary_types.split(",")
+    if labels and isinstance(labels, str):
+        labels = [int(label) for label in labels.split(",")]
+
+    response = client.retrieve_incidents_request(
+        page, items, fromdate, todate, status, adversary_types, labels
+    )
+    command_results = CommandResults(
+        outputs_prefix="Lumu.RetrieveIncidents",
+        outputs_key_field="",
+        outputs=response,
+        raw_response=response,
```

Comment:
add `readable_output` to all the commands

```suggestion
        raw_response=response,
        readable_output=tableToMarkdown('Incidents', response.get('items',[]), headerTransform=pascalToSpace, removeNull=True),
```

Reply:
readable_output were added in commands output  in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7

Reply:
updated in recent commit